database/sql/V072__auction_solution_orders.sql

```diff
@@ -0,0 +1,57 @@
+-- Rename current `auctions` table to `auction` since it always contains only the latest auction
+ALTER TABLE auctions RENAME TO auction;
+
+CREATE TABLE auctions (
+   auction_id bigint PRIMARY KEY,
+   -- The block number at which the auction was created
+   block bigint NOT NULL,
```

Comment:
Added while at it, since already needed for https://github.com/cowprotocol/services/issues/2844

Reply:
How does this table relate to the existing auctions table? Isn't this going to cause a naming conflict?

Reply:
Existing `auctions` table is renamed to `auction` (or should we rename it to `latest_auction`?). 

Not all auctions that appear in `auction` become entry in `auctions`. We don't want to save auctions that had no competition as those are useless.

I agree this isn't perfect and ideally we have one table `auctions` with all historic and the latest auction. But this requires significant changes in how the autopilot and orderbook handle auctions.

Reply:
Update: removed renaming for safety. New table is called `competition_auctions`, unfortunately I don't have a better name ATM.